index.bs

```diff
@@ -134,6 +134,11 @@ handle additional, application-specific use cases.
     DocumentFragment sanitize(SanitizerInput input);
     DOMString sanitizeToString(SanitizerInput input);
 
+    static readonly attribute Sanitizer default;
+    static readonly attribute Sanitizer nofetch;
+    static readonly attribute Sanitizer nonavigate;
```

Comment:
If I may, I'll punt on the discussion of the nofetch & nonavigate use case for now, maybe even for all of v1. I fully agree that we will have to consider this at some point though :)